features/largeVideo/actions.js

```diff
@@ -28,10 +29,14 @@ export function selectEndpoint() {
             const videoTrack = getTrackByMediaTypeAndParticipant(
                 tracks, MEDIA_TYPE.VIDEO, largeVideo.participantId);
 
-            conference.selectParticipant(
-                videoTrack && videoTrack.videoType === VIDEO_TYPE.CAMERA
-                    ? largeVideo.participantId
-                    : null);
+            try {
+                conference.selectParticipant(
```

Comment:
I know it's not introduced by this PR but the fact that this PR had to add a low-level (i.e. features/base) error handler into such a high-level (i.e. non-base) feature as largeVideo to me affirms my feeling: the leaking of lib-jitsi-meet objects such as .jitsiConference which are wrapped by jitsi-meet-react outside of their wrapping feature has probably spread complexity (&, consequently, all negative effects that complexity brings to source code). Anyway.